### DISCUSSION TITLE
NO OFF LINE MODE  2  ???

### DISCUSSION THREAD

**Discussion by alexpi4** [QuestCraftPlusPlus/QuestCraft#233] [General]
Yesterday I was without internet for about 8 hours..
No internet, and no offline mode. it's impossible to play
so if we are without internet I can't play, is that it?

**Comment by essassination** [comment]
The developers removed offline mode to prevent piracy of Minecraft Java Edition.
If you still want to play offline, you must connect to the internet to login (therefore proving your ownership of Minecraft), and then turn your Wi-Fi off.

**Reply by alexpi4** [reply]
But that's the point, my friend! when you have no internet, you can't play the game even if you bought the license. and this is for all games that follow this model of authentication via the internet. We pay for the license to use, but we are hostage to the internet. If it becomes unavailable, we are unable to use the game for which we paid for the license. The moment game companies adopted this authentication model, they should be co-responsible for the availability of the service license we pay for, and they should provide authentication alternatives, and not simply blocking pirated offline access in detriment to paying customers. I have paid for the "license to use" and I want to "use it" any time I want. Have you ever thought about it?

**Reply by essassination** [reply]
Well if you feel strongly about this feature you can request that it be added

**Comment by mikareq** [comment]
Maybe there is an option to prove ownership of Minecraft only once - after installation and then we could play offline?

**Reply by essassination** [reply]
Pretty sure the developers cannot due that because they can't change the minecraft account system. It's just run by microsoft

**Reply by mikareq** [reply]
on pc you log only once and then you can play offline

**Reply by alexpi4** [reply]
> on pc you log only once and then you can play offline

it's not on PC, it's on QUEST 2

**Comment by wellsilver** [comment]
I think this is better suited as an issue for pojav-launcher if it wasnt removed by questcraft
